keyboards/irene/keymaps/default/config.h

```diff
@@ -0,0 +1,10 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
```

Comment:
Please use the `#pragma once` preprocessor directive instead of traditional header guards.